cobigen/cobigen-core-api/pom.xml

```diff
@@ -66,8 +65,23 @@
       <version>2.27.2</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>2.3.1</version>
+    </dependency>
+
+    <!-- JAXB RI -->
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-impl</artifactId>
+      <version>2.3.4</version>
+    </dependency>
```

Comment:
These were already set in our root pom.xml.

Reply:
Unfortunately we can not access them

Reply:
Done